pkgs/top-level/perl-packages.nix

```
@@ -2487,6 +2500,21 @@ let
     };
   };
 
+  ChordPro = buildPerlPackage {
+    pname = "App-Music-ChordPro";
+    version = "0.974";
+    src = fetchurl {
+      url = mirror://cpan/authors/id/J/JV/JV/App-Music-ChordPro-0.974.tar.gz;
+      sha256 = "7fae50535dcb7f68171a8c1f8ad49b13563f406fb1d44eba78b349e7ef83a8af";
+    };
+    propagatedBuildInputs = [ AppPackager FileLoadLines FontTTF IOString ImageInfo PDFAPI2 StringInterpolateNamed ];
+    meta = {
+      homepage = http://www.chordpro.org;
```

```suggestion
      homepage = "http://www.chordpro.org";
```

Quote URLs per https://github.com/NixOS/rfcs/pull/45